### DISCUSSION TITLE
Updatecli new logo

### DISCUSSION THREAD

**Discussion by olblak** [updatecli/updatecli#678] [General]
I created the octopus logo in the rush when I realized that I needed one.
My design skills are what they are, I tried to modify an octopus picture but I could never achieve anything interesting.
Either it's hard to have something that render correctly in different situation such as small picture or in dark mode.
I am not expecting improving my skill any time soon so I experimented with a much simpler version.

@lemeurherve @dduportal what do you think of:

![Updatecli Logo (1)](https://user-images.githubusercontent.com/2360224/166441268-66e68a74-02fa-43ec-928f-daf56f8673de.png)

**Comment by MarckK** [comment]
How about the above logo with the interior of the U a different color, with a matching colored triangle on top?  The  U wraps an arrow pointing up.  

Very exciting project, y'all!  🚀

**Reply by olblak** [reply]
I like the suggestion, let me try something

**Comment by olblak** [comment]
What do you think of 
![image](https://user-images.githubusercontent.com/2360224/167297341-2baf614e-7721-45b9-99d7-8d805ce92b22.png)

**Comment by olblak** [comment]
Or 
![image](https://user-images.githubusercontent.com/2360224/167297435-0a04feb8-cdc0-4a9a-8231-91c3a1b16e12.png)

**Comment by olblak** [comment]
Thanks for the feedback, as @lemeurherve said, we shouldn't be too similar to Udemy business logo
Maybe we could use a different color as well 

I am sharing [here](https://docs.google.com/presentation/d/1Tt0RnpBsZf83w2zAEHl2wfFljLbfi9mESRrucFt1B9w/edit?usp=sharing) the different experimentation that I did

**Comment by olblak** [comment]
Here is another one with a blue color

![image](https://user-images.githubusercontent.com/2360224/167450582-e501d58f-f049-48c6-9a17-4223b3e45f5a.png)

**Reply by lemeurherve** [reply]
Not fan of this one (or the one with 2 "arrrows"), it make me think of a military distinction sign.

**Reply by olblak** [reply]
For me, it makes me thing about  accelerators in video game :D

**Comment by olblak** [comment]
I am going to wait until the end of the week before picking one

**Comment by MarckK** [comment]
Having fun and added more color 😄 
It might also look good if the green color was limited to inside the arrow and all around the U and arrows the dark navy blue color remained.

<img width="767" alt="Screenshot 2022-05-09 at 12 21 43" src="https://user-images.githubusercontent.com/15032115/167454077-41370c1a-4025-4bd3-9d33-ec1f0e31f8bd.png">

**Comment by MarckK** [comment]
Tried it with only arrow green, but somehow the proportions seem off...

<img width="736" alt="Screenshot 2022-05-09 at 14 46 53" src="https://user-images.githubusercontent.com/15032115/167476905-e8f8b883-2685-4c61-b07e-3e16595de724.png">

**Comment by olblak** [comment]
I think, I'll take a bit more time to experiment with a different logo idea

**Comment by olblak** [comment]
Right so another week, and another round of inspiration 

![image](https://user-images.githubusercontent.com/2360224/169964816-b374e54d-77c6-427b-be60-8dd68a79637f.png)

**Comment by olblak** [comment]
![image](https://user-images.githubusercontent.com/2360224/169964891-9da463a5-3c95-4f39-acb0-a9d97dc43c80.png)

**Comment by olblak** [comment]
![image](https://user-images.githubusercontent.com/2360224/169964984-a25d62e7-7650-4a93-99d7-b53f883cd7cc.png)

**Comment by olblak** [comment]
![image](https://user-images.githubusercontent.com/2360224/169965055-62b53e3c-ffcc-4853-a312-203fa90c8fae.png)

**Comment by olblak** [comment]
![image](https://user-images.githubusercontent.com/2360224/169965693-c6209171-4e99-413a-9808-1c54a65d1546.png)

**Comment by olblak** [comment]
After quite some discussion with different people, unless objection, I am going to switch to  https://github.com/updatecli/website/pull/367/files

**Comment by lemeurherve** [comment]
Updated our Github App logo, [looks good](https://github.com/jenkins-infra/kubernetes-management/pull/2403) 😃 
<img width="271" alt="image" src="https://user-images.githubusercontent.com/91831478/170305208-512efe10-e3eb-42bd-ae82-9fc3c720fe07.png">

**Reply by olblak** [reply]
Awesome Thanks for doing that

**Reply by lemeurherve** [reply]
<img width="562" alt="image" src="https://user-images.githubusercontent.com/91831478/170740634-b43bcff5-f4ba-4295-91da-3b9cc14a912b.png">
@olblak you might want to do the same here ;)

**Reply by olblak** [reply]
Nice spot, I just updated the that bot picture 🎉
